### PR TITLE
Personal.md

### PR DESCRIPTION
**Pull request by rgottesman**
Update personal settings page

### CODE REVIEW

File: docs/personal.md [hunk 1]
```diff
@@ -15,10 +15,45 @@ To access your personal settings, click ![Settings cog -icon](assets/images/user
...
+
+To change your password, enter your current password and then enter a new one.
+
+## Multi-factor authentication
+
+Enable multi-factor authentication for an added level of security. 
+
+#### To enable multi-factor authentication:
+
+1. Use your phone to scan the QR code that appears on the screen.
```
Comment (spinningrachel) [inline docs/personal.md:34]:
```suggestion
1. From the Personal Settings area, in the My Profile pane click MFA. 
    
    The Multi-Factor Authentication window loads, similar to the following: 
    
    <insert mfa.png (I already added it to the folder)>
    
2. Toggle the button to enable MFA.

```

File: docs/personal.md [hunk 2]
```diff
@@ -15,10 +15,45 @@ To access your personal settings, click ![Settings cog -icon](assets/images/user
...
+
+	You need to scan the code using an authenticator app like Google Authenticator. 
+
+2. Enter the 6-digit code from your authenticator app.
```
Comment (spinningrachel) [inline docs/personal.md:38]:
```suggestion
2. From Cycode, enter the 6-digit code.
```

File: docs/personal.md [hunk 3]
```diff
@@ -15,10 +15,45 @@ To access your personal settings, click ![Settings cog -icon](assets/images/user
...
+
+3. Save your recovery code.
+
+	If you lose your phone, the recovery code enables you to log in.
```
Comment (spinningrachel) [inline docs/personal.md:42]:
```suggestion
	The recovery code enables you to log in if you lose your phone, 
```

File: docs/personal.md [hunk 4]
```diff
@@ -15,10 +15,45 @@ To access your personal settings, click ![Settings cog -icon](assets/images/user
...
+
+#### To enable multi-factor authentication:
+
+1. Use your phone to scan the QR code that appears on the screen.
+
+	You need to scan the code using an authenticator app like Google Authenticator. 
+
+2. Enter the 6-digit code from your authenticator app.
+
+3. Save your recovery code.
```
Comment (spinningrachel) [inline docs/personal.md:40]:
```suggestion
3. Save your recovery code in a safe place.
```

**Review by spinningrachel — CHANGES_REQUESTED** [review]

File: docs/personal.md [hunk 5]
```diff
@@ -4,9 +4,9 @@ Every user can access their personal settings to configure and manage:
```
Comment (spinningrachel) [inline docs/personal.md:4]: Enter an explanation how to navigate to this area.

File: docs/personal.md [hunk 6]
```diff
@@ -15,10 +15,45 @@ To access your personal settings, click ![Settings cog -icon](assets/images/user
...
+
+	If you lose your phone, the recovery code enables you to log in.
+
+## Access tokens
+
+You automatically have the option to integrate everything shown in [**Integrations**](link to integrations page). If you need to integrate something else, create a personal access token. With these tokens, you authorize other applications to integrate Cycode.  
+
+#### To create a personal access token:
+
+1. Click **Create New**.
```
Comment (spinningrachel) [inline docs/personal.md:50]: @rgottesman try adding results and images to the rest of the processes

**Review by spinningrachel — CHANGES_REQUESTED** [review]

File: docs/personal.md (hunk 6 above)
Comment (rgottesman) [reply docs/personal.md:50]: The personal access token page looks different when there aren't any tokens created yet. Can I delete the token you created, or should I wait until we get a clean account?

File: docs/personal.md [hunk 7]
```diff
@@ -15,10 +15,45 @@ To access your personal settings, click ![Settings cog -icon](assets/images/user
...
 
+- see the name and email address used in your account
 
-## Password
+- change the name in your account
 
-## Multi-factor authentication {#authentication}
+## Change password
+
+To change your password, enter your current password and then enter a new one.
```
[the remark this answers is not in the document]
Comment (rgottesman) [reply docs/personal.md:26]: Is an authenticator app necessary to change passwords also? Or just for MFA (in which case the prerequisite would be in that section)?

[the remark this answers is not in the document]
Comment (rgottesman) [reply docs/personal.md:27]: @spinningrachel ?

**Pull request merged**